salt/config/__init__.py

```diff
@@ -40,14 +41,13 @@
 import salt.exceptions
 from salt.utils.locales import sdecode
 import salt.defaults.exitcodes
+import salt.grains.core
 
 try:
     import psutil
     HAS_PSUTIL = True
 except ImportError:
     HAS_PSUTIL = False
-    import platform
-    import salt.grains.core
```

Comment:
The reason this was here is because we don't want to import core grains unless we absolutely have to.